test_report_with_reconnected_reporter_but_finished_jobs - AssertionError: 10 seconds should be sufficient to send three events

Got this while fixing the truncated normal function:

https://github.com/equinor/ert/actions/runs/6118155838/job/16605864095?pr=6064

```
FAILED tests/unit_tests/job_runner/test_event_reporter.py::test_report_with_reconnected_reporter_but_finished_jobs - AssertionError: 10 seconds should be sufficient to send three events
assert (1694152206.449184 + 10) > 1694152216.4759638
 +  where 1694152216.4759638 = <built-in function time>()
 +    where <built-in function time> = time.time
= 1 failed, 1447 passed, 25 skipped, 1 xfailed, 30 warnings in 198.57s (0:03:18) =
```

--- a/tests/unit_tests/job_queue/test_ert_qstat_proxy.py
+++ b/tests/unit_tests/job_queue/test_ert_qstat_proxy.py
@@ -10,6 +10,7 @@ from pathlib import Path
 
 import pytest
 import testpath
+from flaky import flaky
 
 from ert import _clib
 
@@ -241,6 +242,7 @@ def test_options_passed_through_proxy(tmpdir, options, expected, monkeypatch):
     # (the output from the mocked qstat happens to adhere to json syntax)
 
 
+@flaky
 @pytest.mark.skipif(sys.platform.startswith("darwin"), reason="No flock on MacOS")
 def test_many_concurrent_qstat_invocations(tmpdir, monkeypatch):
     """Run many qstat invocations simultaneously, with a mocked backend qstat

--- a/tests/unit_tests/job_runner/test_event_reporter.py
+++ b/tests/unit_tests/job_runner/test_event_reporter.py
@@ -5,6 +5,7 @@ import time
 from unittest.mock import patch
 
 import pytest
+from flaky import flaky
 
 from _ert_job_runner.client import ClientConnectionClosedOK, ClientConnectionError
 from _ert_job_runner.job import Job
@@ -199,6 +200,7 @@ def test_report_with_failed_reporter_but_finished_jobs(unused_tcp_port):
     assert len(lines) == 0, "expected 0 Job running messages"
 
 
+@flaky
 @pytest.mark.skipif(
     sys.platform.startswith("darwin"), reason="Performance can be flaky"
 )

